verilog/formatting/formatter_test.cc

```diff
@@ -295,6 +295,19 @@ static const std::initializer_list<FormatterTestCase> kFormatterTestCases = {
      "  `FOO( bar )\n",
      "`define FOO BAR\n"
      "`FOO(bar)\n"},
+    {// multiple argument macro call
+     "  `FOO( bar , baz )\n",
+     "`FOO(bar, baz)\n"},
+    {// macro call in function
+     "function void foo( );   foo=`FOO( bar , baz ) ; endfunction\n",
+     "function void foo();\n"
+     "  foo = `FOO(bar, baz);\n"
```

Comment:
Can you also add a few test cases that cover nested macro calls:
`FOO(`BAR(baz1, baz2))`

Reply:
Done.